src/css/custom.css

```diff
@@ -153,3 +153,8 @@ html[data-theme='dark'] main img {
   padding: 1rem;
   border-left: 3px solid var(--ifm-tabs-color-active-border);
 }
+
+/* Style Chrome extension screenshot images */
+.wallet-extension-img {
+  max-width: 350px;
+}
```

Comment:
Is this really needed? I think all images should be managed by Docusaurus default theme CSS. Also, if we decide to use some more fancy light-box like images plugin in the future.